ccloud/docs/beginner-cloud.rst

```diff
@@ -0,0 +1,902 @@
+   in the ACL you created previously:
+
+   .. code-block:: bash
+
+      ccloud kafka acl create --allow --service-account 104349 --operation WRITE --topic '*'
+
+   You should see the following output:
+
+   .. code-block:: text
+
+         ServiceAccountId | Permission | Operation | Resource | Name |  Type
+       +------------------+------------+-----------+----------+------+---------+
+         User:104349      | ALLOW      | WRITE     | TOPIC    | *    | LITERAL
+
+
+#. Run the following command to allow user ``104349`` to read from any topic
+   in the ACL:
+
+   .. code-block:: bash
+
+      ccloud kafka acl create --allow --service-account 104349 --operation READ --topic '*'
+
+   .. code-block:: text
+
+         ServiceAccountId | Permission | Operation | Resource | Name |  Type
+       +------------------+------------+-----------+----------+------+---------+
+         User:104349      | ALLOW      | READ      | TOPIC    | *    | LITERAL
+
+
+#. Run the following command to allow user ``104349`` to read from any topic
+   in the ACL:
+
+   .. code-block:: bash
+
+       ccloud kafka acl create --allow --service-account 104349 --operation READ --consumer-group connect
+
+   .. code-block:: text
+
+         ServiceAccountId | Permission | Operation | Resource |  Name   |  Type
+         +------------------+------------+-----------+----------+---------+---------+
+         User:104349      | ALLOW      | READ      | GROUP    | connect | LITERAL
+
+#. Verify the ACLs were configured by running the following command:
+
+   .. code-block:: bash
+
+      ccloud kafka acl list --service-account 104349
+
+   .. code-block:: text
+
+         ServiceAccountId | Permission | Operation | Resource |  Name   |  Type
+       +------------------+------------+-----------+----------+---------+---------+
+         User:104349      | ALLOW      | WRITE     | TOPIC    | *       | LITERAL
+         User:104349      | ALLOW      | CREATE    | TOPIC    | *       | LITERAL
+         User:104349      | ALLOW      | READ      | TOPIC    | *       | LITERAL
+         User:104349      | ALLOW      | READ      | GROUP    | connect | LITERAL
+
+#. Generate environment variables with |ccloud| connection information for
+   |kconnect| to use:
+
+   .. code-block:: text
+
+      ../../ccloud/ccloud-generate-cp-configs.sh /tmp/client.config &>/dev/null
+      source delta_configs/env.delta
+
+# Run a |kconnect| container with the kafka-connect-datagen plugin:
+
+  .. code-block:: bash
+
+     docker-compose up -d
+
+  You should see the following output:
+
+  .. code-block:: text
+
+      Creating connect-cloud ... done
+      Waiting up to 60 seconds for Docker container for connect to be up
+      ............
+
+#. Post the configuration for the kafka-connect-datagen connector that produces
+   pageviews data to |ccloud| topic ``demo-topic-3``:
+
+   .. code-block:: bash
+
+      curl --silent --output /dev/null -X POST -H "Content-Type: application/json" --data
```

Comment:
Looks like the command was truncated.  Should be:

```
DATA=$( cat << EOF
{ 
  "name": "$CONNECTOR",
  "config": {
    "connector.class": "io.confluent.kafka.connect.datagen.DatagenConnector",
    "kafka.topic": "$TOPIC3",
    "quickstart": "pageviews",
    "key.converter": "org.apache.kafka.connect.storage.StringConverter",
    "value.converter": "org.apache.kafka.connect.json.JsonConverter",
    "value.converter.schemas.enable": "false",
    "max.interval": 5000,
    "iterations": 1000,
    "tasks.max": "1"
  }
}
EOF
)
curl --silent --output /dev/null -X POST -H "Content-Type: application/json" --data "${DATA}" http://localhost:8083/connectors
```

Reply:
Updated